packages/admin-portal/src/core/index.tsx

```diff
@@ -5,10 +5,6 @@ import { render } from 'react-dom'
 import ReactGA from 'react-ga'
```

Comment:
See above, a small refactor of the run function required https://github.com/reapit/foundations/blob/master/packages/aml-checklist/src/core/index.tsx#L61